### PR TITLE
f - 监控模块配置告警接收人时，告警组与轮班组下拉框优化#826

### PR DESCRIPTION
**Pull request by guo-zi-xin**
#### 修复问题
- #826

### CODE REVIEW

**Comment by openc3-bot-001** [comment]
test/ok

build-c3-front:ok
check-yml-file-format:ok
compiler-bl-proxy:ok
compiler-jumpserver:ok
compiler-perl:ok
compiler-trouble-ticketing:ok

**Comment by openc3-bot-001** [comment]
test/ok

build-c3-front:ok
check-yml-file-format:ok
compiler-bl-proxy:ok
compiler-jumpserver:ok
compiler-perl:ok
compiler-trouble-ticketing:ok

**Comment by openc3-bot-001** [comment]
test/ok

build-c3-front:ok
check-yml-file-format:ok
compiler-bl-proxy:ok
compiler-jumpserver:ok
compiler-perl:ok
compiler-trouble-ticketing:ok

File: c3-front/src/app/pages/quickentry/monitorconfig/monitorconfig.html [hunk 1]
```diff
@@ -226,20 +226,28 @@ <h5>{{'C3T.监控策略'|translate}}</h5>
...
-                    <select class="form-control" style="width: 250px" ng-model="newUser"  ng-options="o.name as o.name for o in monitorconfig.monitorgroup"> </select>
-                    <input type="text" style="width: 250px" class="form-control" ng-model="monitorconfig.subgroup"  placeholder="{{'C3T.子组'|translate}}">
-                </div>
-                <button type="submit" ng-disabled="myFormGroup.$invalid" class="btn btn-primary" ng-click="monitorconfig.createUser()"><i class="fa fa-user fa-lg" aria-hidden="true"></i>{{'C3T.添加报警组'|translate}}</button>
-            </form>
+              <ui-select class="form-group" ng-model="newUser" style="width: 250px;">
+                <ui-select-match ><span class="item-label">{{$select.selected.name}}</span></ui-select-match>
+                <ui-select-choices
+                  repeat="item.name as item in monitorconfig.monitoroncall | filter: $select.search">
+                  <div ng-bind-html="item.name | highlight: monitorconfig.monitoroncall"></div>
```
Comment (lijinfeng2011) [inline c3-front/src/app/pages/quickentry/monitorconfig/monitorconfig.html]: 下拉框内容不正确

**Comment by lijinfeng2011** [comment]
修改后无法提交保存

**Pull request merged**